docs/Getting Started/Installation/index.md

```diff
@@ -1,6 +1,10 @@
 # Installation
 In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software development kit (SDK) must be installed in your app. The following steps will guide you through adding the SDL SDK to your workspace and configuring the environment.
 
+!!! NOTE
+The SDL SDK is currently supported on @![iOS]`iOS 8.0`!@@![android] `Android 2.2 (Froyo)`!@@![javaSE, javaEE] `Java 7 (1.7)`!@ and above.
```

Comment:
Can you remove the code tags from around the OS system names? Plain text formatting should be fine.

Reply:
@NicoleYarroch just to be clear, are you referring to removing the `!!! NOTE` tag?

Reply:
The code tags refer to the `...` surrounding the operating system names (i.e. remove the backticks from iOS 8.0, Android 2.2, etc., like so:

The SDL SDK is currently supported on @![iOS]iOS 8.0!@@![android]Android 2.2 (Froyo)!@@![javaSE, javaEE]Java 7 (1.7)!@ and above.

Reply:
Got it, fixed in most recent commit.